crates/sui/src/utils.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
In general I'm wary of `utils`-flavor files because they tend to devolve into dumping grounds rather than "belonging" to a set of functionality. Wondering again if having this logic at the input portion when parsing would help it "belong" better to the pipeline we have for processing aliases versus non-aliases.

Reply:
Agreed with this too, but somehow when I think about this function it feels like it does not belong in the `client_commands` file; on the other hand, that's the only place where it's used so maybe it should stay there. Let's see how I go about the first feedback and then go from there.